bayesapi/resources/associated_columns.py

```diff
@@ -10,6 +10,7 @@ def on_post(self, req, resp):
 
         req_vars = validate(self.api_def, 'post', req)
         target_column = req_vars['column']
+        lower_target_column = target_column.lower()
```

Comment:
Since `lower_target_column` is just two characters shorter than `target_column.lower()` I'd just inline it. If you wanted to keep the variable I'd move it to just before where it's used to reduce cognitive load for the reader.